### DISCUSSION TITLE
iOS performance issues on latest update

### DISCUSSION THREAD

**Discussion by hrfarmer** [ppy/osu#23006] [Q&A]
I'm using an iPhone XR. On the latest version (2023.326.1), using the metal renderer has made performance significantly worse, to the point where I would say the game would be unplayable past 4* max

https://imgur.com/a/i4gmY0y
(on old renderer, performance still wasn't great but i guess its fine for casual gameplay)

https://imgur.com/a/gZ2y1XT
(on metal, literally just unplayable)

also as a fun bonus, changing my renderer from metal to opengl straight up bricked the game, it will not open now.

https://user-images.githubusercontent.com/58487401/227840612-8241384b-6c08-4144-8374-be69ad26a5a0.mov

**Comment by hrfarmer** [comment]
realized i forgot to add logs (in reference to the renderer crash)

[database.log](https://github.com/ppy/osu/files/11074596/database.log)
[input.log](https://github.com/ppy/osu/files/11074597/input.log)
[network.log](https://github.com/ppy/osu/files/11074598/network.log)
[performance.log](https://github.com/ppy/osu/files/11074599/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11074600/runtime.log)

**Comment by peppy** [comment]
Thanks for the report. We've also noticed this and it may have regressed during development. Stay tuned!
